content/post/the-future-is-rusty.en.md

```diff
@@ -0,0 +1,69 @@
++++
+title = 'The Future is Rust(y)?'
+slug = 'post'

```

Comment:
Better slugs man. A slug needs to be descriptive. 
```suggestion
slug = 'future-with-rust-swarnim-arun'
```
How does that look, I can just read the url and tell you what the blog is about.

Reply:
Aah I think I skimmed over it :P